app/js/reducers/discontinueOrderReducer.js

```diff
@@ -0,0 +1,30 @@
+import {
+  DISCONTINUE_ORDER_FAILURE,
+  DISCONTINUE_ORDER_SUCCESS,
+} from '../actions/actionTypes';
+import mockData from '../../../__mocks__/mockData';
+
+const initialState = [
+  mockData.defaultpatientActiveOrder,
+  mockData.pastOrders,
+  mockData.loading,
+];
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case DISCONTINUE_ORDER_SUCCESS:
+      return {
+        ...state,
+        activeOrders: state.activeOrders,
+        pastOrders: state.pastOrders,
+        loading: action.status,
+      };
+    case DISCONTINUE_ORDER_FAILURE:
+      return {
+        ...state,
+        error: action.error,
+      };
+    default:
+      return state;
+  }
```

Comment:
I can't seem to find case for the Loading reducer... `LOADING_DISCONTINUE_ORDER_FAILURE`. Is this intentional?

Reply:
I don't have that as an action type because of the general loader, so if you look at the 2 switch cases, you'll see where they were used